Authentication/SessionServiceType.swift

```diff
@@ -94,7 +94,7 @@ public protocol SessionServiceType {
          Avoid sending a .LogIn event as well.
          Don't forget to update the currentUser.
     */
-    func signUp(name: String, email: Email, password: String) -> SignalProducer<User, SessionServiceError>
+    func signUp(name: String?, email: Email, password: String) -> SignalProducer<User, SessionServiceError>
```

Comment:
I don't know. Isn't it the same? They can use it as any name (username or first name, etc)